src/apiCalls.js

```diff
@@ -0,0 +1,13 @@
+const apiCalls = {
+  getMovieData(id) {
```

Comment:
This is an interesting way to reduce unnecessary network requests. I don't think I see this done usually - so we'll have to talk about this later on to see if you experienced any drawbacks.